app_dart/lib/src/request_handlers/postsubmit_luci_subscription.dart

```diff
@@ -81,6 +88,20 @@ class PostsubmitLuciSubscription extends SubscriptionHandler {
     await datastore.insert(<Task>[task]);
     log.fine('Updated datastore');
 
+    if (task.status == Task.statusFailed || task.status == Task.statusInfraFailure) {
+      log.fine('Trying to auto-retry...');
```

Comment:
Does this also handle the manual retry?

Reply:
`ResetProdTask` is for manual retries, but it uses the same `LuciBuildService.checkRerunBuilder` call (granted it can ignore the default checks)